src/utils/utils.cairo

```diff
@@ -478,4 +478,13 @@ namespace Helpers {
         let (q, _) = unsigned_div_rem(bytes_len + 31, 32);
         return q * 32;
     }
+
+    // @notice Returns the min value between a and b
+    func min{range_check_ptr}(a:felt, b: felt) -> felt {
+        if(is_le(a, b) == 1) {
```

Comment:
@spapinistarkware told us one day that it's better (save some resources) to compare against 0 instead of 1 (FALSE instead of TRUE), so better write this the other way around